workflow/scripts/build_scenarios.py

```diff
@@ -5,39 +5,44 @@
 
 # This script reads in data from the IIASA database to create the scenario.yaml file
 
-import pyam
+
 import ruamel.yaml
 from pathlib import Path
 import pandas as pd
 import os
 
-def get_shares(df):
+def get_shares(df, planning_horizons):
     # Get share of vehicles for transport sector - meglecting heavy duty vehicles
-    total_transport = df.loc["Stock|Transportation|LDV"]
-    tech_transport = df.loc[["Stock|Transportation|LDV|ICE",
-                "Stock|Transportation|LDV|BEV",
-                "Stock|Transportation|LDV|PHEV",]]
+    # At the 
+    total_transport = df.loc["DEMO v1", "Stock|Transportation|LDV"]
+    tech_transport = df.loc["DEMO v1"].loc[[ 
+        "Stock|Transportation|LDV|ICE",
+        "Stock|Transportation|LDV|BEV",
+        "Stock|Transportation|LDV|PHEV",
+    ]]
 
     transport_share = tech_transport / total_transport
-    transport_share = transport_share[[2020, 2025, 2030, 2035, 2040, 2045]]
-    transport_share.reset_index(drop=True, inplace=True)
+    transport_share = transport_share[planning_horizons]
     transport_share.set_index(pd.Index(["ICE", "BEV", "PHEV"]), inplace=True)
 
     # Get share of Navigation fuels
-    total_navigation = df.loc[["Final Energy|Bunkers|Navigation",
-                         "Final Energy|Transportation|Domestic Navigation"]].sum(axis=0)
-    navigation_liquid = df.loc[["Final Energy|Bunkers|Navigation",
-                         "Final Energy|Transportation|Domestic Navigation|Liquids"]].sum()
-    navigation_h2 = df.loc[["Final Energy|Transportation|Domestic Navigation|Hydrogen"]]    
+    total_navigation = \
+        df.loc["REMIND-EU v1.1", "Final Energy|Bunkers|Navigation"] + \
```

Comment:
Is there a reason why you use two different models for that? If so maybe leave a comment